pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -80,6 +80,11 @@ let
         url = "https://github.com/python/cpython/pull/8985.patch";
         sha256 = "1c8nq2c9sjqa8ipl62hiandg6a7lzrwwfhi3ky6jd3pxgyalrh97";
       })
+      # CVE-2018-14647. See https://bugs.python.org/issue34623
+      (fetchpatch {
+        url = "https://github.com/python/cpython/commit/18b20bad75b4ff0486940fba4ec680e96e70f3a2.patch";
```

Comment:
https://github.com/python/cpython/commit/10be1d3f802b874914b2a13eb41407c7a582d9b3.patch is the patch that's merged in the 2.7 branch.

Reply:
That commit only adds the news entry. I've used the commit which includes the fix, it's also referenced in the Debian tracker https://security-tracker.debian.org/tracker/CVE-2018-14647